NimicClientTest/NimicClientTest.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
The name of the file and folder is wrong... or is this intended for C#?

Reply:
Ok, I think it's intended haha, but it looks a bit like a typo to me.

Reply:
Yes, it's for the .Net [family of programming languages](https://en.wikipedia.org/wiki/List_of_CLI_languages), which includes C# among others. The client only depends on the package `System.Text.Json` which is available for [.Net Standard 2.0](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-overview) and should be supported for [all platforms](https://dotnet.microsoft.com/platform/dotnet-standard).